independent-publisher-connectors/AzureADApplications/apiDefinition.swagger.json

```diff
@@ -49,23 +54,60 @@
             "x-ms-summary": "Search criteria",
             "x-ms-visibility": "advanced"
           },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Filters results (rows)",
+            "x-ms-summary": "Filter criteria",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "ConsistencyLevel",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "Set this property to 'eventual' if you are using $search",
+            "x-ms-summary": "Enable advanced queries",
+            "x-ms-visibility": "advanced",
+			"enum": [

```

Comment:
@paulculmsee all files should be properly formatted. may I ask you please to fix the indentation for all the files.